tests/Unit/Facade.Tests/SaCoreTypesServiceSpecs/WhenUpdatingSaCoreType.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace Linn.Products.Facade.Tests.SaCoreTypesServiceSpecs
+{
+    using FluentAssertions;
+
+    using Linn.Common.Facade;
+    using Linn.Products.Domain.Linnapps;
+    using Linn.Products.Resources;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenUpdatingSaCoreType : ContextBase
+    {
+        private IResult<SaCoreType> result;
+
+        private SaCoreTypeResource resource;
+
+        [SetUp]
+        public void SetUp()
+        {
+            
+            this.resource = new SaCoreTypeResource()
+                                {
+                                    CoreType = 1,
+                                    Description = "new desc",
+                                    DateInvalid = "1995/03/28",
```

Comment:
As discussed - I'd make all the resource dates use the ISO format for more realistic testing.